lux/test/unit/lux/prisms/twitter/get_tweet_test.exs

```diff
@@ -0,0 +1,220 @@
+defmodule Lux.Prisms.Twitter.GetTweetTest do
+  @moduledoc """
+  Test suite for the GetTweet prism module.
+
+  These tests verify the prism's ability to:
+  - Retrieve tweets from Twitter by ID
+  - Handle expanded fields like author and metrics
+  - Process Twitter API errors appropriately
+  - Validate input/output schemas
+
+  The tests use a Twitter API client mock to simulate API interactions.
+  """
+
+  use UnitAPICase, async: true
+  alias Lux.Prisms.Twitter.GetTweet
+  alias Lux.Test.Mocks.TwitterClientMock
+
+  @tweet_id "1234567890"
+  @tweet_text "This is a test tweet from Lux"
+  @author_id "9876543210"
+  @author_username "test_user"
+  @agent_ctx %{name: "TestAgent"}
+
+  setup do
+    # Set up the TwitterClientMock
+    Application.put_env(:lux, Lux.Integrations.Twitter.Client, plug: {Req.Test, TwitterClientMock})
```

Comment:
Please let's do this in UnitAPICase since all unit tests using twitter should use Req.Test.